exchanges/account/account_test.go

```diff
@@ -314,6 +330,12 @@ func TestBalanceInternalLoad(t *testing.T) {
 	if bi.GetFree() != 3 {
 		t.Fatal("unexpected value")
 	}
+
+	err = bi.load(&Balance{UpdatedAt: now, Total: 2, Hold: 3, Free: 4, AvailableWithoutBorrow: 5, Borrowed: 6})
+	assert.Error(t, err, "should have not been loaded")
```

Comment:
🚧 Can we please check the error value?
```suggestion
	assert.ErrorIs(t, errOutOfSequence, err, "should error correctly with same UpdatedAt")
```